pkg/web/web.go

```diff
@@ -14,13 +14,13 @@
 package web
 
 import (
+	"log/slog"
 	"net/http"
 	"net/url"
 
-	"github.com/go-kit/log"
 	"github.com/prometheus/client_golang/prometheus"
+	clientVersion "github.com/prometheus/client_golang/prometheus/collectors/version"
```

Comment:
Minor nit, we have been using the alias `versioncollector`.

Reply:
hrm, looks like influxdb/graphite exporter use this form still :thinking: 

```
./influxdb_exporter/main.go:31: clientVersion "github.com/prometheus/client_golang/prometheus/collectors/version"
./graphite_exporter/cmd/graphite_exporter/main.go:28:   clientVersion "github.com/prometheus/client_golang/prometheus/collectors/version"
```

happy to standardize on `versioncollector`, though :+1:

Reply:
Yea, there's a few that deviate.